docs_src/design/TOC.md

```diff
@@ -7,27 +7,28 @@
+| [0005 Service Self Config Init](./adr/0005-Service-Self-Config.md)                          | Service Self Config Init & Config Seed Removal  |
+| [0006 Metrics Collection](./adr/0006-Metrics-Collection.md)                                 | Collection of service telemetry data |
+| [0007 Release Automation](./adr/devops/0007-Release-Automation.md)                          | Overview of Release Automation Flow for EdgeX   |
+| [0008 Secret Distribution](./adr/security/0008-Secret-Creation-and-Distribution.md)         | Creation and Distribution of Secrets                    |
+| [0009 Secure Bootstrapping](./adr/security/0009-Secure-Bootstrapping.md)                    | Secure Bootstrapping of EdgeX                           |
+| [0011 Device Service REST API](./adr/device-service/0011-DeviceService-Rest-API.md)         | The REST API for Device Services in EdgeX v2.x          |
+| [0012 Device Service Filters](./adr/device-service/0012-DeviceService-Filters.md)           | Device Service event/reading filters  |
+| [0013 Device Service Events via Message Bus](./adr/013-Device-Service-Events-Message-Bus.md) | Device Services send Events via Message Bus             |
+| [0014 Secret Provider for All](./adr/014-Secret-Provider-For-All.md)                        | Secret Provider for All EdgeX Services                  |
+| [0015 Encryption between microservices](./adr/security/0015-in-cluster-tls.md)              | Details conditions under which TLS is or is not used    |
+| [0016 Container Image Guidelines](./adr/security/0016-docker-image-guidelines.md)           | Documents best practices for security of docker images  |
+| [0017 Securing access to Consul](./adr/security/0017-consul-security.md)                    | Access control and authorization strategy for Consul    |
+| [0018 Service Registry](./adr/0018-Service-Registry.md)                                     | Service registry usage for EdgeX services               |
+| [0019 EdgeX-CLI V2](./adr/core/0019-EdgeX-CLI-V2.md)                                        | EdgeX-CLI V2 Implementation                             |
+| [0020 Delay start services (SPIFFE/SPIRE)](./adr/security/0020-spiffe.md)                   | Secret store tokens for delayed start services          |
+| [0021 Device Profile Changes](./adr/core/0021-Device-Profile-Changes.md)                    | Rules on device profile modifications          |
+| [0022 Unit of Measure](./adr/core/0022-UoM.md)                                              | Unit of Measure|
+| [0023 North South Messaging](./adr/0023-North-South-Messaging.md)                           | Provide for messaging from north side systems through command down to device services |
+| [0024 System Events](./adr/0024-system-events.md)                                    | Provide for messaging from north side systems through command down to device services |
```

Comment:
Need to correct the short description in the TOC

Reply:
Good catch, was focused on getting the link correct and forgot about the description. ;-)

Reply:
Fixed